### DISCUSSION TITLE
Synthesia learning with RTP device

### DISCUSSION THREAD

**Discussion by AnnNeveu** [onlaj/Piano-LED-Visualizer#387] [Q&A]
Hi ! I'm quite lost as to how I should connect the Synthesia device to the RPi.
The learning thing ot the visualizer works perfectly, so I have the keys highlighted, waiting for my input. But I'd like to also have the next keys coming up with Synthesia. The thing is, I'm unsure about how I should go about it.

Those are my settings. The RTP port is selected in the web interface (which makes the learning thing don't work, so I know it's connected as it should), and I tried putting both input and output in the RTP app. The IP address is the address of my RPi. Both the RPi and tablet are connected to the same network.

So I really don't know what I'm supposed to do - any help would be greatly appreciated !

![Screenshot_20221002_180828_com awisman midiconnectorfree](https://user-images.githubusercontent.com/90454929/193464257-a93d84ec-eeef-4202-9e81-0b943f307616.jpg)
![Screenshot_20221002_180754_com synthesia synthesia](https://user-images.githubusercontent.com/90454929/193464260-e97db0a8-24d8-492d-997e-8ec8a7f257c3.jpg)

**Comment by TheGuyFawkes** [comment]
Hello!! Were you able to solve this?

**Comment by TheGuyFawkes** [comment]
> Hi ! I'm quite lost as to how I should connect the Synthesia device to the RPi. The learning thing ot the visualizer works perfectly, so I have the keys highlighted, waiting for my input. But I'd like to also have the next keys coming up with Synthesia. The thing is, I'm unsure about how I should go about it.
> 
> Those are my settings. The RTP port is selected in the web interface (which makes the learning thing don't work, so I know it's connected as it should), and I tried putting both input and output in the RTP app. The IP address is the address of my RPi. Both the RPi and tablet are connected to the same network.
> 
> So I really don't know what I'm supposed to do - any help would be greatly appreciated !
> 
> ![Screenshot_20221002_180828_com awisman midiconnectorfree](https://user-images.githubusercontent.com/90454929/193464257-a93d84ec-eeef-4202-9e81-0b943f307616.jpg) ![Screenshot_20221002_180754_com synthesia synthesia](https://user-images.githubusercontent.com/90454929/193464260-e97db0a8-24d8-492d-997e-8ec8a7f257c3.jpg)

Sorry to insist, did you manage to fix it? I'm going crazy trying to get the LEDs to work in training mode. I even bought the MIDI Hub for Android but I still can't get it :(
